ui/test/layouts/document/test/nuxeo-test-edit-layout.html

```diff
@@ -0,0 +1,40 @@
+<!--
+`nuxeo-test-edit-layout`
+@group Nuxeo UI
+@element nuxeo-test-edit-layout
+-->
+<dom-module id="nuxeo-test-edit-layout">
+  <template>
+    <style>
+      *[role='widget'] {
+        @apply --nuxeo-widget;
+      }
+    </style>
+
+    <nuxeo-input role="widget" name="Title" label="Title" value="{{document.properties.dc:title}}" autofocus required>
+    </nuxeo-input>
+
+    <nuxeo-textarea role="widget" name="Description" label="Description" value="{{document.properties.dc:description}}">
+    </nuxeo-textarea>
+  </template>
+
+  <script>
+    Polymer({
+      is: 'nuxeo-test-edit-layout',
+      behaviors: [Nuxeo.LayoutBehavior],
+      properties: {
+        /**
+         * @doctype Test
+         */
+        document: Object,
+      },
+
+      validate() {
+        if (this.document.properties['dc:description'] === this.document.properties['dc:title']) {
```

Comment:
`if (cond) { return false; } return true` is the same as `return !cond`

Reply:
Will do. I just wanted to make something really dumb and clear, show that it must return a boolean. :P